spec/jobs/private_body_decrypt_job_spec.rb

```diff
@@ -2,28 +2,33 @@
 
 require "spec_helper"
 
-describe "rake decidim:set_decrypted_private_body", type: :task do
+describe "rake decidim:private_body_decrypted_job", type: :task do
```

Comment:
In the`spec/jobs/private_body_decrypt_job_spec.rb` we expect the described class to be `PrivateBodyDecryptJob`, can you update this spec file to test the `perform` method please ?

Reply:
Oh Yes of course 👍